github/PullRequest.py

```diff
@@ -837,12 +837,30 @@ def is_merged(self):
         status, headers, data = self._requester.requestJson("GET", self.url + "/merge")
         return status == 204
 
+    def restore_branch(self):
+        self.head.repo.create_git_ref("refs/heads/" + self.head.ref, sha=self.head.sha)
```

Comment:
Just return create_git_ref, we can deal with its return value. This function also needs a docstring.

Reply:
You are just saying: 
```suggestion
        return self.head.repo.create_git_ref("refs/heads/" + self.head.ref, sha=self.head.sha)
```